lib/Db/BoardMapper.php

```diff
@@ -89,13 +91,20 @@ public function find($id, $withLabels = false, $withAcl = false) {
 	}
 
 	public function findAllForUser(string $userId, int $since = -1, $includeArchived = true): array {
-		$groups = $this->groupManager->getUserGroupIds(
-			$this->userManager->get($userId)
-		);
-		$userBoards = $this->findAllByUser($userId, null, null, $since, $includeArchived);
-		$groupBoards = $this->findAllByGroups($userId, $groups,null, null, $since, $includeArchived);
-		$circleBoards = $this->findAllByCircles($userId, null, null,  $since, $includeArchived);
-		return array_unique(array_merge($userBoards, $groupBoards, $circleBoards));
+		if (!array_key_exists($userId, $this->userBoardCache)) {
```

Comment:
`isset()` is said to be faster, but there is a small difference –https://www.php.net/manual/en/function.array-key-exists.php example #2. For consideration.